docs/sphinx/source/api.rst

```diff
@@ -97,6 +97,8 @@ Filtering
    :toctree: generated/
     
    clip_filter
+   logic_clip_filter
+   xgboost_clip_filter
```

Comment:
Also add `tune_clip_filter_plot` to the `Plotting` section.

Side note, I wonder if `clip_filter` ought to be renamed to `quantile_clipping_filter` or something...